keps/sig-testing/2290-new-label-for-trusted-PR-identification/kep.yaml

```diff
@@ -1,19 +1,25 @@
 title: New label for trusted PR identification
 kep-number: 2290
 authors:
-  - "@matthyx"
+  - matthyx
 owning-sig: sig-testing
 participating-sigs:
   - sig-contributor-experience
 reviewers:
-  - "@fejta"
-  - "@cjwagner"
-  - "@BenTheElder"
-  - "@cblecker"
-  - "@stevekuznetsov"
+  - fejta
```

Comment:
I wonder if we can automate normalizing this (parse and spit back out?) and stick it in a linter /shrug

Reply:
`kepctl` could totally do this, I just didn't want to change the world while there are ~100 PR's outstanding